cli/storage_profile.c

```diff
@@ -414,16 +405,15 @@ int
 hse_storage_profile(const char *path, bool quiet, bool verbose)
 {
     struct storage_info info = {0};
-    int                i, rc;
-    int                thread_counts[] = { 16, 20, 24, 32, 40, 48 };
-    const int          num_thread_counts = sizeof(thread_counts) / sizeof(thread_counts[0]);
-    double             scores[num_thread_counts];
-    double             avg_score;
-    int                cnt_score;
-    uint64_t           max_thrds = thread_counts[num_thread_counts - 1];
-    uint64_t           max_space_needed;
-    const char        *result = "heavy";
-    char               pathbuf[PATH_MAX];
+    struct itimerval timer = {0};
```

Comment:
If you feel the desire to order these from longest to smallest...

Reply:
Please don't, that put the longest definitions nearest the first line of code, much better to sort the other way around which makes the guts of the funciton easier to read when it's not crowded by a bunch of definitions.

Reply:
I try to order it first by type and then from longest to shortest. In this case, my hands are tied a bit because of the data dependency among the variables.